cvat/requirements/base.txt

```diff
@@ -52,4 +52,4 @@ azure-storage-blob==12.8.1
 # --no-binary=pycocotools: workaround for binary incompatibility on numpy 1.20
 # of pycocotools and tensorflow 2.4.1
 # when pycocotools is installed by wheel in python 3.8+
-datumaro==0.1.9 --no-binary=datumaro --no-binary=pycocotools
+git+https://github.com/manasars/datumaro.git@cvat-3D-M6-pcd --no-binary=datumaro --no-binary=pycocotools
```

Comment:
@manasars , could you please fix the link and point to the develop branch of Datumaro?

Reply:
Done.